cardano-api-gen/src/Gen/Cardano/Api/Typed.hs

```diff
@@ -562,7 +563,7 @@ genTxBodyContent era = do
 
   pure $ TxBodyContent
     { txIns = map (, BuildTxWith (KeyWitness KeyWitnessForSpending)) trxIns
-    , txInsCollateral = TxInsCollateralNone --TODO: Alonzo era: Generate collateral inputs.
+    , txInsCollateral = txInCollateral'
```

Comment:
We could simplify this function using named field puns.

Reply:
How does it look now?